### ISSUE TITLE
[Feature Request] Switch for doorbell activation

### ISSUE THREAD

**Issue by wapster182** [lukasroegner/homebridge-nuki#69]
Hi,

I would love an on/off switch for the doorbell in Homekit.
Currently, it is ringing no matter if I by myself using the Ring to open function or I am home and somebody else is ringing. In the first case the ringing should not be necessary. With a switch I could automate the disabling of the ringing by geofencing.
Or maybe even a function directly within the plugin? So that the homekit doorbell does not trigger on ring to open ringings.
That would be so great!

**Comment by Tyraenor** [comment]
Yes, I really would like to see that, too. That is the only reason, why I cannot enable it, because I would wake my wife, if I ring to open myself in, if she has the early shift.

**Comment by Nastras** [comment]
@lukasroegner The extension would be great 😉

**Comment by Nastras** [comment]
Since I have long heard nothing more from you I wanted to ask how the chances are that you implement it soon?

LG

**Comment by lukasroegner** [comment]
Hi @Nastras, 
Sadly, there is not enough time at the moment to develop new features for the homebridge plugins. I hope that will change soon.

**Comment by Nastras** [comment]
Hey Lukas,
nice to hear from you. Thought you were lost 😁

That you currently have no time for your plugins is a pity but there are more important things. Still good to know that you are still there and sometime when the time allows it to develop your plugins further.

I wish you a happy Easter and let me hear from you from time to time 😉

Best greetings
